### DISCUSSION TITLE
How to control and change the crystal orientation in 2D?

### DISCUSSION THREAD

**Discussion by ehsanmoshkelgosha22** [neperfepx/neper#243] [Q&A] [answered]
Hi,

It seems that the Crystal Orientation Options are working for the 3D domain. Is it possible to control and change the crystal orientation in 2D? How is it possible to have a 2D polycrystalline domain with random crystal orientation? Is it possible to implement our own orientations file in 2D?

**Comment by rquey** [comment, marked as answer]
Yes, it works just as well in 2D as in 3D...

For "2D" orientations (rotations only in the x-y plane), use `-ori "fiber(0,0,1,0,0,1)"`.
